### ISSUE TITLE
Something about the attention_mask setting

### ISSUE THREAD

**Issue by elevenofji** [abhilash1910/LongPegasus#3]
Hi, I'm just interested in your work. I'm trying to turn BART into longformer version. In your test program, you just use the pegasus tokenizer to tokenize the sentence,and don't care about the "attention_mask", right? 
I mean, In the huggingface's longformer version, "-1,0,1" means "no attention, local attention and global attention", but it is different from the definition of bart tokenizer (which 1 means attention, 0 means no attention.  So does pegasus?). So when it comes to the "attention_mask", the situation may be more complicated, right? The user has to define the attention_mask first.

**Comment by abhilash1910** [comment]
Hi @elevenofji ,
Yes you are correct. For longformer the implication of -1,0,1 are correct; and I do not require the attention mask for the tokenization.  But if you want to convert BART to longformer version , you can directly refer to the source present [here](https://github.com/allenai/longformer/blob/master/scripts/convert_bart_to_longformerencoderdecoder.py). For the masking part, most of the logic is the same for both bart and pegasus ; you can find that from the HF implementation [here](https://github.com/huggingface/transformers/blob/master/src/transformers/models/pegasus/modeling_tf_pegasus.py).  Considering both of them have the same base structure (with differences in layer numbers and beams) , it has the same 1/0 logic and you can optionally define the attention mask there as well . Since pegasus mainly aims to solve summarization; whereas Bart can be used for certain masked tasks such as Conditional modelling/generation,paraphrasing , you can use the attention mask as well .

**Comment by abhilash1910** [comment]
Hi @elevenofji ,
Closing due to inactivity. Kindly reopen if any issue is found.
